spec.emu

```diff
@@ -390,6 +407,31 @@ contributors: Google, Ecma International
     </emu-note>
   </emu-clause>
 
+  <emu-clause id="sup-temporal-calendardateextendedyear" type="abstract operation">
+    <h1>
+      CalendarDateExtendedYear (
+        _calendar_: a String,
+        _date_: a Temporal.PlainDateTime, Temporal.PlainDate, or Temporal.PlainYearMonth,
+      ): an integer
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by _calendar_ relative to a well-defined epoch year for that calendar.</dd>
+    </dl>
+    <p>
+        This definition supersedes the definition provided in "Temporal proposal", #sec-temporal-calendardateextendedyear.
+    </p>
+    <emu-alg>
+      1. Let _year_ be _date_.[[ISOYear]].
+      1. If IsCalendarSupportEra(_calendar_) is *false*, then
+        1. Return _year_.
```

Comment:
This will need to be renamed to match your other PR that I merged earlier. Although, `iso8601` is the only calendar for which it would return false, and this op should never be called with `iso8601`. So maybe a better and clearer option would be to turn it into an assertion. That is,
- change the type of the _calendar_ parameter to "a calendar type that is not **"iso8601"**"
- change this step to "Assert: CalendarSupportsEra(_calendar_) is **true**."